kythe/go/indexer/cmd/go_indexer/go_indexer.go

```diff
@@ -104,9 +105,19 @@ func main() {
 		docURL = u
 	}
 
+	visit := indexGo
+	if *contOnErr {
```

Comment:
I don't know go style much.  What's the tradeoff to doing this vs having indexGo return a function?

(putting the logic inside indexGo instead of out here, and then calling visitPath(ctx, path, indexGo(*contOnErr)))

Reply:
It's simpler to just handle the logic inside of indexGo.